content/800-data-platform/300-data-proxy.mdx

```diff
@@ -239,6 +240,18 @@ Each Data Proxy instance has its own connection pool. The size of each of these
 postgresql://user:password@host:5432/mydb?connection_limit=60
 ```
 
+### Idle connections to the database
+
+The Data Proxy creates several connection pools that maintain open idle connections even when no queries run from your application. The connection pools and any remaining idle connections close after 6 minutes of inactivity (no queries are sent to the database).
+
+In each of the supported locations, the Data Proxy architecture includes multiple clusters that maintain idle connections to the database to ensure high availability in case of failures in any of the clusters.
```

Comment:
Consider splitting this sentence into two to simply it. Maybe something like:

"... to the database. These idle connections ensure..."

Also, would "to your database" feel a bit more direct than "to the database"?